docs/Configuration.rst

```diff
@@ -2908,46 +2910,43 @@ These Portal caches can be overridden in
 ::
 
     # Portal Cache Configuration - MOP session Manager
-    #  - Standalone (live time in seconds)
-    exo.cache.portal.mop.maxsize=5000
-    exo.cache.portal.mop.livetime=600
-    #  - Cluster (expiration in milliseconds)
-    exo.cache.portal.mop.maxnodes=5000
-    exo.cache.portal.mop.expiration=600000
+    exo.cache.portal.mop.MaxNodes=5000
+    exo.cache.portal.mop.TimeToLive=-1
+    exo.cache.portal.mop.strategy=LIRS
+    # For Cluster mode
+    exo.cache.portal.mop.cacheMode=replication
```

Comment:
The cache mode of caches config are not good, see https://github.com/exodev/platform/blob/develop/extension/config/src/main/resources/conf/platform/configuration.properties#L689 for the good default values.

Reply:
(y)